src/index.ts

```diff
@@ -10,3 +10,4 @@ export { getKafka } from './initializers/kafka';
 export { getRabbit } from './initializers/rabbitmq';
 export { getRedis, createRedisConnection } from './initializers/redis';
 export { middlewares };
+export { validateBody, validateQueryString } from './initializers/koa/validate-params';
```

Comment:
Could this be added in the middlewares folder? I think it matches with middlewares.

Reply:
I'm not opposed to it, but what about riviere and add-request-id? Are they not considered middlewares as well?

Reply:
Moreover, should I add the errors folder under middlewares as well or leave them in koa folder?

Reply:
Those 2 are not exposed to the client. They are internal.

Reply:
So let's leave them where they are for now.

Reply:
Done

Reply:
Were they moved to middelewares?